packages/react-theme/src/alias/darkColor.ts

```diff
@@ -141,22 +141,3 @@ export const generateColorTokens = (brand: BrandVariants): ColorTokens => ({
   colorBrandShadowAmbient: 'rgba(0,0,0,0.30)', // rgba(0,0,0,0.30) undefined
   colorBrandShadowKey: 'rgba(0,0,0,0.25)', // rgba(0,0,0,0.25) undefined
 });
-
-export const colorPaletteTokens: ColorPaletteTokens = (Object.keys(sharedColors) as Array<
```

Comment:
`colorPaletteTokens` have been moved to separate files to clearly separate files generated by the pipeline from the manually maintained files.
With the next pipeline update, the generated files will contain `/* GENERATED */` comment.